mrmustard/lab_dev/circuits.py

```diff
@@ -427,6 +430,45 @@ def validate_path(self, path) -> None:
 
             remaining[i1] = (remaining[i1] @ remaining.pop(i2))[0]
 
+    def serialize(self, filestem=None):
+        r"""
+        Serialize a Circuit.
+
+        Keyword Args:
+            filestem (str): An optional name to give the resulting file saved to disk.
+
+        All components must implement the following in order to serialize correctly:
```

Comment:
Hmm not sure if this needs to be mentioned in the docstring? It almost reads like the docstring of a hidden method (not exposed to the user)

Reply:
Maybe move to the docstring of the CC serialize?